src/index.js

```diff
@@ -105,6 +125,8 @@ const init = (settings) => {
     rows: options.rows,
     hl: options.hl,
     mainQueryField: options.mainQueryField,
+    siteList: options.siteList,
```

Comment:
Can you walk me through where this is used? Based on this config option, I would expect to see some changes to the solr client itself in a different pr, is that the case?

Reply:
This is legacy from the original approach and now safe to remove.